skema/program_analysis/CAST/matlab/tests/test_assignment.py

```diff
@@ -37,3 +37,6 @@ def test_single_assignment():
     assert module['left']['val']['name'] == 'x'
     assert module['right']['value_type'] == 'Integer'
     assert module['right']['value'] == '5'
+
+if __name__ == "__main__":
```

Comment:
This is more of a curiosity, but do you have to have these 'if main' checks for the tests to run on your end? 
When I write my test scripts I don't have to put these in, since pytest just collects every function that starts with a 'test' and runs that way. When I want to run them myself I just do 'pytest FILE' and that executes any functions that start with test in FILE (whatever FILE is).